WordPress/Tests/WP/DeprecatedParametersUnitTest.inc

```diff
@@ -0,0 +1,64 @@
+<?php
+
```

Comment:
Are these groups in order of release? i.e. functions with deprecated params in 4.9 can be added to the end of the relevant groups? If so, maybe add a comment here to remind future contributors.